### PR TITLE
Create, Mint and Transfer NFTs with the Avalanche Wallet

### PR DESCRIPTION
**Pull request by Devilla**
- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://app.gitbook.com/@figment-learn/s/learn-docs/other/tutorial-guidelines).

In particular:

- [x] I've followed the [template structure](Structure).
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've ran a linter & prettifier to format my code blocks.
- [x] I've thoroughlly explained the code, with inline code comments and text explanations.
- [x] I've ran the tutorial and the code works.

### CODE REVIEW

**Comment by Devilla** [comment]
@pyr0gan updated as per requirements. Thanks.

**Review by pyr0gan — CHANGES_REQUESTED** [review]
Thank you for resubmitting, I believe we can merge once you complete these changes.

File: network-documentation/avalanche/tutorials/create-mint-transfer-nft.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+description: Learn how to create, mint, and transfer NFTs on Avalanche.
+---
+
+# Create, Mint and Transfer NFTs with Avalanche Wallet
+
+## Prerequisites
```
[the remark this answers is not in the document]
Comment (Devilla) [reply network-documentation/avalanche/tutorials/create-mint-transfer-nft.md]: okay

File: network-documentation/avalanche/tutorials/create-mint-transfer-nft.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+## Summary
+
+Now, you should know how to create NFT families, mint NFT groups, and send them to other addresses. Have fun with it! If you would like to know the technical background of how NFTs work on the Avalanche network or would like to build products using NFTs, please check out the [NFT tutorial](https://learn.figment.io/network-documentation/avalanche/tutorials/creating-an-nft-part-1).
+
+If you had any difficulties following this tutorial or simply want to discuss Avalanche tech with us you can [**join our community today**](https://discord.gg/fszyM7K)!
+
+## About the author  
+
+[Devendra Yadav](https://github.com/devilla)
```
[the remark this answers is not in the document]
Comment (Devilla) [reply network-documentation/avalanche/tutorials/create-mint-transfer-nft.md:150]: Is it optional? @pyr0gan

**Review by pyr0gan — APPROVED** [review]
Thank you for fixing the image links!

**Comment by Devilla** [comment]
> Thank you for fixing the image links!

pleasure!

**Pull request merged**